### PR TITLE
Add Pax lmspmd2b and lmtransformeradam nightly tests

### PR DESCRIPTION
**Pull request by RissyRan**
# Description

Add Pax lmspmd2b and lmtransformeradam nightly tests:
* Move helper config from `jax` folder to `pax`
* Add helper `get_setup_cmds()` and `get_runtime_version()` in pax config to re-use the `get_pax_lm_config()`
* Add test dependency accordingly (please let me know if you have any questions on this)
* Other small changes

# Tests

Please describe the tests that you ran on Cloud VM to verify changes.

**Instruction and/or command lines to reproduce your tests:** ...
Upload to Airflow and run tests

**List links for your tests (use go/shortn-gen for any internal link):** ...
Currently nightly build failed, so I hardcode the date to 20231203, and got the same error in [old platform](https://b.corp.google.com/issues/308011763#comment18):
* [pax_nightly_lmspmd2b_v4_8](https://screenshot.googleplex.com/BuQ5D4DewDf5D9G)
* [pax_nightly_lmspmd2b_ckpt_v4_8](https://screenshot.googleplex.com/rmwcyfudCja7PfM)
* [pax_nightly_lmtransfoormeradam_v4_8](https://screenshot.googleplex.com/5S775qAQvLFxjvo)
* [pax_nightly_lmtransformeradam_v4_16](https://screenshot.googleplex.com/53DgUeedWqcpUbM)

# Checklist

Before submitting this PR, please make sure (put X in square brackets):
- [x] I have performed a self-review of my code.
- [x] I have necessary comments in my code, particularly in hard-to-understand areas.
- [x] I have run one-shot tests and provided workload links above if applicable. 
- [x] I have made or will make corresponding changes to the doc if needed.

### CODE REVIEW

File: configs/xlml/pax/solutionsTeam_pax_supported_config.py
Comment (gkroiz) [inline configs/xlml/pax/solutionsTeam_pax_supported_config.py:1]: nit: can we keep file naming consistent? (either using snake or camel case, but not both).

Comment (RissyRan) [reply configs/xlml/pax/solutionsTeam_pax_supported_config.py:1]: Thanks Gerson! Good call! I was also thinking about naming consistency on both files and test names. Can I do that in a separate PR? I will need to update all `solutionsTeam` to `solutionsteam` in the repo.

Comment (gkroiz) [reply configs/xlml/pax/solutionsTeam_pax_supported_config.py:1]: Yes, separate PR makes sense

**Review by gkroiz — APPROVED** [review]

File: configs/xlml/pax/solutionsTeam_pax_supported_config.py
Comment (RissyRan) [reply configs/xlml/pax/solutionsTeam_pax_supported_config.py:1]: Sounds good. Will send out that PR in a min.

**Pull request merged**